mod-source-record-storage-server/src/test/java/org/folio/rest/impl/SourceRecordApiTest.java

```diff
@@ -1661,6 +1676,51 @@ private void returnSpecificMarcSourceRecordOnGetByRecordExternalId(TestContext t
     async.complete();
   }
 
+  private void returnSpecificMarcSourceRecordOnGetByRecordExternalIdAndRecordState(TestContext testContext,
```

Comment:
can we use Parameterized test here?

Reply:
Good idea, but it requires to add new dependency. And all tests in this module are written that way.
So our test will be different or we have to rewrite all tests